rules/integrations/aws/execution_new_terms_ec2_instance_cloudformation_createstack.toml

```
@@ -0,0 +1,61 @@
+[metadata]
+creation_date = "2020/07/25"
+integration = ["aws"]
+maturity = "production"
+updated_date = "2024/07/25"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule detects the first time a principal calls AWS Cloudwatch `CreateStack` or `CreateStackSet` API. Cloudformation is used to create a single collection of cloud resources called a stack, via a defined template file. An attacker with the appropriate privileges could leverage Cloudformation to create specific resources needed to further exploit the environment. This is a new terms rule that looks for the first instance of this behavior in the last 10 days for a role or IAM user within a particular account.
+"""
+false_positives = [
+    """
+    Verify whether the user identity should be using the `CreateStack` or `CreateStackSet` APIs. If known behavior is causing false positives, it can be exempted from the rule. The "history_window_start" value can be modified to reflect the expected frequency of known activity within a particular environment.
+    """,
+]
+from = "now-6m"
+index = ["filebeat-*", "logs-aws.cloudtrail-*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "AWS IAM User or Role Created Cloudformation Stack for First Time"
```

```suggestion
name = "First Time AWS Cloudformation Stack Creation by User"
```

Does that make sense, if not feel free to ignore!